administration/src/bp-modules/PasswordInput.tsx

```diff
@@ -21,8 +21,13 @@ const ShowPasswordButton = (props: { hidden: boolean; onClick: () => void }) =>
 const PasswordInput = ({
   label,
   setValue,
+  readonly = false,
   ...otherProps
-}: InputGroupProps2 & { label: string; setValue: (value: string) => void }): ReactElement => {
+}: InputGroupProps2 & {
+  label: string
+  setValue: (value: string) => void
+  readonly?: boolean
```

Comment:
:upside_down_face: We could also make this optional and derive `readonly` from it:
```suggestion
  setValue: null | (value: string) => void
```
Below: 
``` tsx
readOnly={setValue === null}
```